### PR TITLE
Create and test function-space-after; closes #81

### LINKED ISSUE #81: New rule: function-space-after

### PR DESCRIPTION
**Pull request by davidtheclark**

### CODE REVIEW

File: src/rules/function-space-after/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ */
+export default function (expectation) {
+  return function (css, result) {
+    css.eachDecl(function (decl) {
+      if (decl.prop === "content") { return }
+
+      const value = decl.value
+
+      for (let i = 0, l = value.length; i < l; i++) {
+        if (value[i] === ")") {
```
Comment (MoOx) [inline src/rules/function-space-after/index.js]: Did you consider using [balanced-match](https://www.npmjs.com/package/balanced-match) for this kind of test ?

Comment (MoOx) [inline src/rules/function-space-after/index.js]: I am thinking about an possible issue with `content: ")";`

Comment (davidtheclark) [inline src/rules/function-space-after/index.js]: No, did not think about looking for something like that (thanks for pointing it out)... At the same time, I'm not sure if this current approach would have bugs that using something like balanced-match would fix. Can you think of any tests that might show that it is necessary, and just looking for closing parens is not enough?

Comment (davidtheclark) [inline src/rules/function-space-after/index.js]: But that's covered with `if (decl.prop === "content") { return }` right?

Comment (MoOx) [inline src/rules/function-space-after/index.js]: what about `content: attr(data-whatever)`? :)

Comment (davidtheclark) [inline src/rules/function-space-after/index.js]: Hadn't thought about that --- yeah, maybe balanced-match would be important, to make sure we get stuff like that and also make sure that we ignore things within strings (e.g. `content: "look at this attr(data-whatever)"`). Could you make an issue for that?

Comment (MoOx) [inline src/rules/function-space-after/index.js]: Sure !

**Comment by MoOx** [comment]
good to merge as is for me, I opened #85 to handle the edge case

**Pull request merged**